consensus/state_sync/module.go

```diff
@@ -36,7 +36,7 @@ type StateSyncModule interface {
 	IsServerModEnabled() bool
 	EnableServerMode()
 
-	SendStateSyncMessage(*typesCons.StateSyncMessage, cryptoPocket.Address, uint64) error
```

Comment:
It's really weird that we're passing a string here.

1. Can you name the parameters so it's clearer to the reader
2. Let's see if we can get rid of the string